apps/prairielearn/src/components/Navbar.html.ts

```diff
@@ -0,0 +1,853 @@
+            </div>
+          </li>
+
+          <li
+            class="nav-item ${navPage === 'instance_admin' && navSubPage === 'assessments'
+              ? 'active'
+              : ''}"
+          >
+            <a class="nav-link" href="${urlPrefix}/instance_admin/assessments">Assessments</a>
+          </li>
+
+          <li
+            class="nav-item ${navPage === 'instance_admin' && navSubPage === 'gradebook'
+              ? 'active'
+              : ''}"
+          >
+            <a class="nav-link" href="${urlPrefix}/instance_admin/gradebook">Gradebook</a>
+          </li>
+
+          ${assessment_label != null && assessment != null
+            ? html`
+                <li class="navbar-text mx-2 no-select">/</li>
+                <li class="nav-item btn-group">
+                  <a
+                    class="nav-link ${navPage === 'assessment' ? 'active' : ''}"
+                    href="${urlPrefix}/assessment/${assessment.id}"
+                  >
+                    ${assessment_label}
+                  </a>
+                  ${assessments != null
+                    ? html`
+                        <a
+                          class="nav-link dropdown-toggle dropdown-toggle-split"
+                          href="#"
+                          id="navbarDropdownMenuLink"
+                          role="button"
+                          data-toggle="dropdown"
+                          aria-haspopup="true"
+                          aria-expanded="false"
+                          aria-label="Change assessment"
+                        >
+                        </a>
+                        <div
+                          class="dropdown-menu"
+                          aria-labelledby="navbarDropdownMenuLink"
+                          id="navbarDropwdownMenuInstructorAssessment"
+                        >
+                          ${assessments.map(
+                            (a) => html`
+                              <a
+                                class="dropdown-item ${navPage === 'assessment' &&
+                                assessment.id === a.id
+                                  ? 'active'
+                                  : ''}"
+                                href="${urlPrefix}/assessment/${a.id}${navPage === 'assessment' &&
+                                navSubPage !== 'file_edit'
+                                  ? `/${navSubPage}`
+                                  : ''}"
+                              >
+                                ${a.assessment_label}
+                              </a>
+                            `,
+                          )}
+                        </div>
+                      `
+                    : ''}
+                </li>
+              `
+            : ''}
+        `
+      : html`
+          <li class="navbar-text mx-2 no-select">/</li>
+
+          <li class="nav-item dropdown" id="navbar-course-instance-switcher">
+            <a
```

Comment:
Can this just be a button?

Reply:
Indeed it was possible, though it required the additional class `btn`. Done in 1edf45b16 for all links with href="#".